kubectl-minio/cmd/tenant-create.go

```diff
@@ -98,15 +99,14 @@ func newTenantCreateCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	f.StringVar(&c.tenantOpts.PrometheusSidecarImage, "prometheus-sidecar-image", "", "(Only used when enable-prometheus is on) The Docker image to use for prometheus sidecar")
 	f.StringVar(&c.tenantOpts.PrometheusImage, "prometheus-init-image", "", "(Only used when enable-prometheus is on) Defines the Docker image to use as the init container for running prometheus")
 	f.BoolVarP(&c.output, "output", "o", false, "generate tenant yaml for 'kubectl apply -f tenant.yaml'")
-
-	cmd.MarkFlagRequired("servers")
-	cmd.MarkFlagRequired("volumes")
-	cmd.MarkFlagRequired("capacity")
-	cmd.MarkFlagRequired("namespace")
```

Comment:
This will be now validated in `TenantOptions.Validate` that way interactive flag can be processed and keep having these flags as required